language.md

```diff
@@ -1023,6 +1023,13 @@ accumulator, which will be returned, which is initially set to _initialexpr_.
 For every item, _expr_ is evaluated with _a_ set to the previously returned
 value.
 
+#### Single Value
+- `Single` _expr_ `Default` _defaultexpr_
+
+Evaluates all _expr_ for each item in the list and returns it if all are the same.
```

Comment:
Looking at `Single`, I'd expect it to behave something like `First` (but somehow different), but that's not the case. What do you think about renaming it to `Same` or `IfSame`?

Reply:
Yeah, I'm not sure what to call it. I don't like `IfSame` because there's no if/else structure. Looking around:

- `Univalued`
- `Always`

There's already a `Single` in the grouping operation, so we should rename both since they do basically the same thing.

Reply:
I like `Univalued`, and could be happy with `Always`